src/backend/InvenTree/build/models.py

```diff
@@ -459,30 +460,30 @@ def is_overdue(self):
         return query.exists()
 
     @property
-    def active(self):
+    def active(self) -> bool:
         """Return True if this build is active."""
         return self.status in BuildStatusGroups.ACTIVE_CODES
 
     @property
-    def tracked_line_items(self):
+    def tracked_line_items(self) -> QuerySet:
         """Returns the "trackable" BOM lines for this BuildOrder."""
         return self.build_lines.filter(bom_item__sub_part__trackable=True)
 
-    def has_tracked_line_items(self):
+    def has_tracked_line_items(self) -> bool:
         """Returns True if this BuildOrder has trackable BomItems."""
         return self.tracked_line_items.count() > 0
 
     @property
-    def untracked_line_items(self):
+    def untracked_line_items(self) -> bool:
         """Returns the "non trackable" BOM items for this BuildOrder."""
         return self.build_lines.filter(bom_item__sub_part__trackable=False)
```

Comment:
This type seems wrong

Reply:
Thanks, fix incoming in https://github.com/inventree/InvenTree/pull/10417